compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -724,38 +710,106 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                 }
             }
         }
-
-        (err, candidates)
     }
 
-    fn detect_assoct_type_constraint_meant_as_path(&self, base_span: Span, err: &mut Diagnostic) {
```

Comment:
Why did you move this method?  This makes review more complicated.

Reply:
This method keep the same except postions difference after code refactored,
https://github.com/chenyukang/rust/blob/code-refactor/compiler/rustc_resolve/src/late/diagnostics.rs#L305

Yes, if most part of the function changed, the diff is not easy to review.

Reply:
Could you make 1 commit per method? Just say so if it's not practical.

Reply:
It's hard now, 
since I didn't make it that way from the starting of refctoring😂

Reply:
Ok, so let's leave it as is :smile:.